build.fsx

```diff
@@ -129,18 +144,23 @@ type ServerPaketDependency = Saturn | Giraffe | Suave
 
 type ServerPaketDependencies =
     { Server : ServerPaketDependency
-      Remoting : bool
+      Communication : Communication option
       Azure : bool }
 
     with override x.ToString () =
             let server = string x.Server
-            let remoting = if x.Remoting then "remoting" else "noremoting"
+            let communication =
+                x.Communication
+                |> Option.map (function
+                    | Remoting -> "remoting"
+                    | Bridge -> "bridge")
+                |> Option.defaultValue "nocommunication"
             let azure = if x.Azure then "azure" else "noazure"
-            sprintf "%s-%s-%s" server remoting azure
+            sprintf "%s-%s-%s" server communication azure
 
 type CombinedPaketDependencies =
     { Azure : bool
-      Remoting : bool
+      Communication : Communication option
```

Comment:
Have you tried running `UpdatePaketLockFiles` after this change? We need to make sure it still works

Reply:
I believe I did, I had to change some lines so the files that weren't created wouldn't fail the build target, then I reverted after creating everything and updated the files.